src/services/ChallengeService.js

```diff
@@ -1209,7 +1209,7 @@ async function update (currentUser, challengeId, data, isFull) {
       if (!_.get(challenge, 'legacy.pureV5Task') && _.isUndefined(_.get(challenge, 'legacy.directProjectId'))) {
```

Comment:
@ThomasKranitsas @rootelement should we now validate the `projectId` ( it might not be needed at all) instead of `legacy.directProjectId`?

Reply:
The `legacy.directProjectId` is used to see if the challenge was created on legacy. We now use the `projectId` property to get the billing account information